javascript/javascript3/week3/searchingSongs.js

```diff
@@ -0,0 +1,76 @@
+
+//Searching for a song
+const songDatabase = [{
```

Comment:
i'm a bit confused, i'm seeing the song search for the 3rd time now. Will skip this file.